datasets/eli5_category/README.md

```diff
@@ -0,0 +1,201 @@
+---
+annotations_creators:
+- found
+language_creators:
+- found
+languages:
+- en
+licenses:
+- unknown
+multilinguality:
+- monolingual
+paperswithcode_id: eli5-category
```

Comment:
I think it doesn't exist on PWC:
https://paperswithcode.com/dataset/eli5-category returns "Not Found"

```suggestion
paperswithcode_id: null
```